dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -564,6 +572,97 @@ public final void fireBulkActions(@Context final HttpServletRequest request,
         }
     }
 
+    @POST
+    @Path("/contentlet/actions/_bulkfire")
+    @JSONP
+    @Produces(SseFeature.SERVER_SENT_EVENTS)
+    public EventOutput fireBulkActions(@Context final HttpServletRequest request,
+            final FireBulkActionsForm fireBulkActionsForm) {
+        final InitDataObject initDataObject = this.webResource
+                .init(null, request, new EmptyHttpResponse(), true, null);
+
+        final EventOutput eventOutput = new EventOutput();
```

Comment:
I think this is ok but eventually you can encapsulate this on an API or Delegate such as
```
ServerSideEventDelegate delegate = SystemEventsAPI.createServerSideEventDelegate();
// .....
delegate.sendMap(map("success", delta));
// .....
delegate.sendString("failed:" + inode));
```